packages/atomic/cypress/fixtures/test-fixture.ts

```diff
@@ -78,6 +79,14 @@ export class TestFixture {
     return this;
   }
 
+  public withTranslation(translations: Record<string, string>) {
```

Comment:
Same as [`test(atomic): cypress test for multi value text
`](https://github.com/coveo/ui-kit/pull/1293/files#diff-28f32bf87b61cb6c20088527208ffdd60b987e5b6d21543f4039c7b4a5ce3bf9R82)